docs/pages/management/operations/backup-restore.mdx

```diff
@@ -239,3 +239,33 @@ of storage backend.
 </TabItem>
 </Tabs>
 
+### Active-passive Multi-region architecture example
```

Comment:
- I'm not sure the backup restore page is the right place for this (it's certainly not where I'd look)
- Do we really recommend active-passive for mission-critical use cases? How many customers do we have running such an architecture?

Reply:
@zmb3 would you be willing and able to suggest where you _would_ look for this kind of content?

Reply:
We do have active/passive users in the wild.  Generally, these are the same users interested in https://github.com/gravitational/teleport/issues/16793 and are looking to have the shortest possible downtime for regional failures as Teleport is mission critical